### PR TITLE
WIP: New ports for opensearch / opensearch-dashboards

### LINKED ISSUE #1: Review and cleannup the WIP commits
Can you please check the commits in the opensearch branch and tell me if you think something is wrong?

I would like to squash them as we move on

### PR DESCRIPTION
**Pull request by smortex**

### CODE REVIEW

File: textproc/opensearch/Makefile [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+	${RM} ${WRKSRC}/lib/jna-*.jar
+
+do-install:
+	${MKDIR} ${STAGEDIR}${PREFIX}/etc/opensearch
+.for f in ${CONFIG_FILES}
+	${INSTALL} ${WRKSRC}/config/${f} ${STAGEDIR}${ETCDIR}/${f}.sample
+.endfor
+	${MKDIR} ${STAGEDIR}${PREFIX}/lib/opensearch/bin
+.for f in ${BINS}
+	${INSTALL_SCRIPT} ${WRKSRC}/bin/${f} ${STAGEDIR}${PREFIX}/lib/opensearch/bin
```
Comment (smortex) [inline textproc/opensearch/Makefile:67]: Is there a reason not install in `$PREFIX/bin`?

File: textproc/opensearch-dashboards/Makefile [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+WRKSRC=		${WRKDIR}/${PORTNAME}-${PORTVERSION}
+NO_BUILD=	yes
+
+do-install:
+	${MKDIR} ${STAGEDIR}${WWWDIR} ${STAGEDIR}${ETCDIR}
+	${INSTALL_DATA} ${WRKSRC}/config/opensearch_dashboards.yml ${STAGEDIR}${ETCDIR}/opensearch_dashboards.yml.sample
+	(cd ${WRKSRC} && \
+		${COPYTREE_SHARE} "data node_modules package.json plugins src" ${STAGEDIR}${WWWDIR} && \
+		${COPYTREE_BIN} bin ${STAGEDIR}${WWWDIR})
```
Comment (smortex) [inline textproc/opensearch-dashboards/Makefile:61]: Is there a reason not install in `$PREFIX/bin`? (This was copied from kibana)

**Review by hackacad — APPROVED** [review]

**Review by hackacad — APPROVED** [review]

**Review by hackacad — APPROVED** [review]

**Comment by smortex** [comment]
Squashed and rebased!  Ready for commit :shipit:

**Comment by smortex** [comment]
Push to the FreeBSD ports tree:
https://cgit.freebsd.org/ports/commit/?id=7fe660d7479c2a88f1c3aac267470f2a507757a0
https://cgit.freebsd.org/ports/commit/?id=19e156d5142115dcd542d47ee329a2656798d789

**Pull request closed without merging**